doc/lua_api.txt

```diff
@@ -3943,7 +3943,9 @@ Definition tables
         diggable = true, -- If false, can never be dug
         climbable = false, -- If true, can be climbed on (ladder)
         buildable_to = false, -- If true, placed nodes can replace this node
-        floodable = false, -- If true, liquids flow into and replace this node
+        floodable = false, --[[
+        ^ If true, liquids flow into and replace this node. However, liquids
+        ^ cannot themselves be floodable. ]]
```

Comment:
I think that the engine ignores the floodable property if the node is also defined as a liquid? At least in my test I could not get water to flow into lava (after marking lava floodable). The on_flood callback wasn't getting called at all (tested by writing to chat inside the callback).